apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -380,14 +380,16 @@ export class WorkshopForm extends React.Component {
     );
   }
 
-  // Returns whether today is within a month before the given session.
+  // Returns whether today is within a month before the given session (a month
+  // is represented as 30 days here for consistency and to align with behavior
+  // in workshop_controller.rb).
   sessionStartsWithinMonth = session => {
     const today = this.props.today;
     const workshopDate = new Date(session.date);
     const monthBeforeWorkshopDate = new Date(
       workshopDate.getFullYear(),
-      workshopDate.getMonth() - 1,
-      workshopDate.getDate()
+      workshopDate.getMonth(),
+      workshopDate.getDate() - 30
```

Comment:
hmm I am not sure this does what we want:
![Screenshot 2023-05-31 at 10 44 23 AM](https://github.com/code-dot-org/code-dot-org/assets/8001765/9f1eab1f-50c9-464e-8fd4-2e42282b6dd2)

Reply:
apologies, this does look like it is working, I just didn't realize the months are 0-indexed (while the days are 1-indexed). this makes it look like it is working properly to me: 
![Screenshot 2023-05-31 at 10 48 43 AM](https://github.com/code-dot-org/code-dot-org/assets/8001765/4ad924ef-1c29-480b-aa00-f39ab70210fd)

Reply:
I think it is right? The month `05` maps to June (0-indexed), and May has 31 days so June 15 - 30 days is May 16?

Reply:
makes sense, thanks for thinking it through!